bundles/org.openhab.core.ui.icon/src/test/java/org/openhab/core/ui/icon/internal/IconServletTest.java

```diff
@@ -74,6 +76,9 @@ public void reset() {
     private IconServlet servlet;
     private ByteArrayServletOutputStream responseOutputStream = new ByteArrayServletOutputStream();
 
+    private @Mock HttpContext httpContext;
```

Comment:
Don't you want to call them mock here?

```suggestion
    private @Mock HttpContext httpContextMock;
```

Reply:
Good catch, I'll update them!